packages/batch-submitter/src/batch-submitter/batch-submitter.ts

```diff
@@ -17,12 +18,21 @@ export interface ResubmissionConfig {
   gasRetryIncrement: number
 }
 
+interface BatchSubmitterMetrics {
+  ethBalance: Gauge<string>
+  batchSizeInBytes: Histogram<string>
+  numTxPerBatch: Histogram<string>
+  minutesSinceLastSubmission: Gauge<string>
```

Comment:
This seems like an odd thing to be measuring... I feel like we ought to be Historgram-ing `submissionTimestamp`?